content/docs/conceptual-guides/branching.md

```diff
@@ -1,21 +1,56 @@
 ---
 title: Branching
+enableTableOfContents: true
 redirectFrom:
   - /docs/conceptual-guides/branches
 ---
 
 <a id="branches-coming-soon/"></a>
 
-Neon Branching capabilities are not yet publicly available. If you would like to try this feature, reach out to [iwantbranching@neon.tech](iwantbranching@neon.tech) describing your use case and requesting that Neon enable branching capabilities for your account.
+Neon allows you to instantly branch your data in the same way that you branch your code. You can quickly and cost-effectively branch your data for development, testing, staging, and various of other purposes, enabling you to improve developer productivity and optimize continuous integration and delivery (CI/CD) pipelines. See [Branching workflows](#branching-workflows) for a discussion of different ways you can integrate branching into your development workflows.
 
-A branch is a copy of the project data created from the current state or any past state that is still available. A branch can be independently modified from its originating project data.
+_Neon Branching capabilities are not yet publicly available. If you would like to try this feature, reach out to [iwantbranching@neon.tech](mailto:iwantbranching@neon.tech) describing your use case and requesting that Neon enable branching for your account._
 
-You can use a branch to:
+## What is a branch?
 
-- Run potentially destructive queries without impacting your main branch
-- Run time travel queries against historical state
-- Run a set of queries with separate resources to avoid impacting your application
-- Tag and name the current moment for PITR convenience or ML model training repeatability
-- Run tests against a branch from production data
+A branch is a copy-on-write clone of your data. You can create a branch from a current or past state. For example, you can create a branch that includes all data up to the current point in time or up to a past point in time or [Log Sequence Number (LSN)](../../reference/glossary#lsn).
```

Comment:
not sure the link `../../reference/glossary#lsn` resolves for me!
do we have that header in glossary?